bpf/bpf_lxc.c

```diff
@@ -173,26 +188,17 @@ static __always_inline int handle_ipv6_from_lxc(struct __ctx_buff *ctx, __u32 *d
 	/* No hairpin/loopback support for IPv6, see lb6_local(). */
 #endif /* ENABLE_PER_PACKET_LB */
 
-	tuple->nexthdr = ip6->nexthdr;
-	ipv6_addr_copy(&tuple->daddr, (union v6addr *)&ip6->daddr);
-	ipv6_addr_copy(&tuple->saddr, (union v6addr *)&ip6->saddr);
-
-	hdrlen = ipv6_hdrlen(ctx, &tuple->nexthdr);
-	if (hdrlen < 0)
-		return hdrlen;
-
-	l4_off = ETH_HLEN + hdrlen;
-
-	/* Pass all outgoing packets through conntrack. This will create an
-	 * entry to allow reverse packets and return set cb[CB_POLICY] to
-	 * POLICY_SKIP if the packet is a reply packet to an existing incoming
-	 * connection.
```

Comment:
Is there any information in this comment that's worth keeping?

Reply:
I don't think so. It's not even correct anymore.